libs/ngx-mime/src/lib/viewer/recognized-text-content/recognized-text-content.component.scss

```diff
@@ -7,5 +7,11 @@
   }
 }
 ::ng-deep .selectedHit {
-  background: rgba(255, 225, 0, 0.6);
+  background: rgba(255, 138, 0, 0.61);
```

Comment:
Is this ment to be the same color as `#ff8900` just with opacity?
`#ff8900` uses 137 as green value instead of 138. If meant to be the same color they should use the same values.